landingpage/templates/landingpage/index.html

```diff
@@ -0,0 +1,115 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block title %}
+    {% trans 'Welcome to Hopper!' %}
+{% endblock title %}
+
+{% block content %}
+<div class="row">
+  <div class="large-12 columns">
+    <div class="row">
+      <div class="large-12 hide-for-small">
+        <div id="featured" data-orbit>
+          <img src="http://placehold.it/1200x500/46CFAF/486070/&text=Slide Image 1" alt="slide image">
+          <img src="http://placehold.it/1200x500/46CFAF/486070/&text=Slide Image 2" alt="slide image">
+          <img src="http://placehold.it/1200x500/46CFAF/486070/&text=Slide Image 3" alt="slide image">
```

Comment:
Please replace ampersand with question mark like in the [example](http://placeholder.in/examples).